tools/cldr-code/src/test/java/org/unicode/cldr/unittest/NistUnits.java

```diff
@@ -27,36 +31,139 @@
-    static final Splitter SPLIT_COMMAS = Splitter.on(',').trimResults();
-    static final Splitter SPLIT_PARENS = Splitter.on('(').trimResults();
+    // HACK for temperature
+    /**
+     * degree Celsius (°C) kelvin (K) T/K = t/°C + 273.15 degree centigrade 15 degree Celsius (°C)
+     * t/°C ≈ t/deg. cent. degree Fahrenheit (°F) degree Celsius (°C) t/°C = (t/°F - 32)/1.8 degree
+     * Fahrenheit (°F) kelvin (K) T/K = (t/°F + 459.67)/1.8 degree Rankine (°R) kelvin (K) T/K =
+     * (T/°R)/1.8 kelvin (K) degree Celsius (°C) t/°C = T/K - 273.15
+     */
+    static final Map<String, Rational> temperatureHack =
+            ImmutableMap.of(
+                    "fahrenheit|celsius", Rational.of("-32/1.8"),
+                    "fahrenheit|kelvin", Rational.of("459.67/1.8"),
+                    "celsius|kelvin", Rational.of("273.15"));
 
     static {
+        Multimap<String, String> _idChanges = LinkedHashMultimap.create();
+        Set<String> _skipping = new LinkedHashSet<>();
+        List<ExternalUnitConversionData> _externalConversionData = new ArrayList<>();
+        Multimap<String, String> _unitToQuantity = TreeMultimap.create();
+        Map<String, TargetInfo> unitToTargetInfo = new TreeMap<>();
+        Map<String, ExternalUnitConversionData> _unitToData = new TreeMap<>();
+        Set<String> _siAcceptable = new TreeSet<>();
+
+        load(
+                _externalConversionData,
+                _unitToQuantity,
+                unitToTargetInfo,
+                _idChanges,
+                _skipping,
+                _unitToData,
+                _siAcceptable);
+
+        skipping = ImmutableSet.copyOf(_skipping);
+        idChanges = ImmutableMultimap.copyOf(_idChanges);
+        externalConversionData = ImmutableSortedSet.copyOf(_externalConversionData);
+        unitToData = ImmutableMap.copyOf(_unitToData);
+        unitToQuantity = ImmutableMultimap.copyOf(_unitToQuantity);
+        derivedUnitToConversion = ImmutableMap.copyOf(unitToTargetInfo);
+        SiAcceptable = ImmutableSet.copyOf(_siAcceptable);
+        if (DEBUG) {
+            for (ExternalUnitConversionData item : externalConversionData) {
+                System.out.println(item);
+            }
+        }
+    }
+
+    static void load(
+            List<ExternalUnitConversionData> _externalConversionData,
+            Multimap<String, String> _unitToQuantity,
+            Map<String, TargetInfo> unitToTargetInfo,
+            Multimap<String, String> _idChanges,
+            Set<String> _skipping,
+            Map<String, ExternalUnitConversionData> _unitToData,
+            Set<String> _siAcceptable) {
         try {
-            Multimap<String, String> _idChanges = LinkedHashMultimap.create();
-            Set<String> _skipping = new LinkedHashSet<>();
+            // Get the SI acceptable units
+            // Unfortunately, this page has inconsistent formats, so we just mine it for
+            // the systems
```

Comment:
might be good feedback… 

Reply:
Not sure how to get it to them... Ideas?